src/mappers/validation-failed-error-mapper.js

```diff
@@ -0,0 +1,134 @@
+'use strict';
+
+/**
+ * Module dependencies.
+ */
+
+const { ValidationFailedError } = require('@uphold/http-errors');
+const { isArray, isEmpty, isUndefined, map, mapValues } = require('lodash');
```

Comment:
We're currently favouring not destructuring lodash, since methods such as `get` or `map` sound generic enough that in the code it's not always obvious they refer to lodash.